### PR TITLE
COM-67: Adapt Redirects-DataGrid-Column Sizes

### PR DESCRIPTION
**Pull request by jennyvivid**
image

### CODE REVIEW

**Comment by johnnyomair** [comment]
It seems to me that this column widths are too big. What would happen on a screen that only has ~1000px width?

File: packages/admin/cms-admin/src/redirects/RedirectsTable.tsx [hunk 1]
```diff
@@ -80,7 +80,7 @@ export function RedirectsTable({ linkBlock, scope }: Props): JSX.Element {
             field: "source",
             headerName: intl.formatMessage({ id: "comet.pages.redirects.redirect.source", defaultMessage: "Source" }),
             sortable: true,
-            flex: 1,
```
Comment (thomasdax98) [inline packages/admin/cms-admin/src/redirects/RedirectsTable.tsx:83]: You can use different values for `flex`: https://mui.com/x/react-data-grid/column-dimensions/#fluid-width

So you could use `flex: 3` here for example

File: packages/admin/cms-admin/src/redirects/RedirectsTable.tsx [hunk 2]
```diff
@@ -80,7 +80,7 @@ export function RedirectsTable({ linkBlock, scope }: Props): JSX.Element {
...
+            minWidth: 600,
```
Comment (thomasdax98) [inline packages/admin/cms-admin/src/redirects/RedirectsTable.tsx:83]: We also want to support mobile screens in the future. So this `minWidth` is too big. Maybe go down to 300 or 400

File: packages/admin/cms-admin/src/redirects/RedirectsTable.tsx [hunk 3]
```diff
@@ -99,6 +99,7 @@ export function RedirectsTable({ linkBlock, scope }: Props): JSX.Element {
             },
             sortable: false,
             flex: 1,
+            minWidth: 400,
```
Comment (thomasdax98) [inline packages/admin/cms-admin/src/redirects/RedirectsTable.tsx:102]: Here you could also set flex to 2 and reduce the minWidth (maybe 200)

File: packages/admin/cms-admin/src/redirects/RedirectsTable.tsx [hunk 4]
```diff
@@ -107,6 +108,7 @@ export function RedirectsTable({ linkBlock, scope }: Props): JSX.Element {
             renderCell: (params) => <div>{params.value}</div>,
             sortable: false,
             flex: 1,
+            minWidth: 400,
```
Comment (thomasdax98) [inline packages/admin/cms-admin/src/redirects/RedirectsTable.tsx:111]: Same as above

File: packages/admin/cms-admin/src/redirects/RedirectsTable.tsx [hunk 5]
```diff
@@ -107,6 +108,7 @@ export function RedirectsTable({ linkBlock, scope }: Props): JSX.Element {
...
             filterable: false,
         },
         {
```
Comment (thomasdax98) [inline packages/admin/cms-admin/src/redirects/RedirectsTable.tsx:114]: I just checked on the mobile screen and assigning `flex: 1` to `generationType`, `active` and `actions` squeezes them on mobile (the field is smaller than its content). So you could remove the `flex` prop entirely for these columns. Then they use as much space as their content needs.

**Review by thomasdax98 — COMMENTED** [review]
Once you are done with a PR, please remove the draft status. Also, @johnnyomair should always be assigned as reviewer.

**Comment by johnnyomair** [comment]
Why do you need `minWidth` at all? Shouldn't `flex` be enough? The min widths still add up to 800 pixels, which is definitely larger than a mobile device. But perhaps we should focus on mobile in a follow-up PR.

**Comment by thomasdax98** [comment]
We probably need a condensed view for mobile. I would do that in a follow-up PR.

**Review by thomasdax98 — DISMISSED** [review]

**Review by thomasdax98 — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**